bom/application/pom.xml

```diff
@@ -623,6 +623,16 @@
                 <artifactId>quarkus-devservices-common</artifactId>
                 <version>${project.version}</version>
             </dependency>
+            <dependency>
+                <groupId>io.quarkus</groupId>
+                <artifactId>quarkus-oidc-devservices-keycloak</artifactId>
```

Comment:
@stuartwdouglas 
`quarkus-oidc-keycloak` sounds good to me, what about the suggestion to have it all in `quarkus-oidc` - please see my comment above.  
So 2 options - we have either 1) `quarkus-oidc` and Keycloak Dev Services/UI are enabled via the property or 2) we have `quarkus-oidc-keycloak`. 

If we have  `quarkus-oidc-keycloak` then it would be closer to a zero effort idea of enabling Dev Services, but I'm not sure now again, I think `quarkus-oidc` only with Keycloak users having to add one property is not bad but now sure if you agree - let me know please - I think both options have their pros and minor cons, I'm fine with either option at this stage, just let me know what you prefer please

Reply:
Hey Stuart, @stuartwdouglas, I've come the full circle in my understanding how it should be packaged and after all these tries and knowing I can put it all into `deployment` without affecting the final jar size, your original idea of having it all in `quarkus-oidc` can work well for all the variations. Let me summarize, hopefully this PR will be ready to go soon, so:

1. if `quarkus:dev` is started with the empty `application.properties` then it is reasonable to default to KC specific Dev Services/UI experience as the user has not indicated which provider is required - so OOB the `OIDC`card will link to the `Keycloak` page.
2. The user will disable it by doing `quarkus.oidc.devservices.enabled=false` - I think there should be no `keycloak` in this property name, keycloak specific configuratiions would be in the `quarkus.oidc.devservices.keycloak.` namespace. 
 2.1 If non-Keycloak users will push back against having to disable it to avoid the Keycloak containers starting in their case OOB with the empty properties then I think we will just switch the default value - or we can revisit the `quarkus-oidc-keycloak` idea, lets see how it goes.
3. if `quarkus.oidc.auth-server-url` is already set then the KC container won't start either
4. if no KC container is started then only an OidcDebugger like Dev UI will be provided where the tokens would be acquired from Keycloak/Okta/Auth0/Google/etc using the standard grants/flows
5. if Quariverse will have `quarkus-oidc-okta`/`quarkus-oidc-auth0`/etc dev services extensions then they will override the default OidcDebugger like/ or Keycloak specific Dev Services/UI provided by `quarkus-oidc`
  
I think it all now kind of clicks and should work fine for all the variations, I'll start working on collapsing it all inside `quarkus-oidc` tomorrow morning - unless you'll object by then :-)

thanks

Reply:
@stuartwdouglas It is all in `quarkus-oidc` now, I'll be tuning it next